src/index.css

```diff
@@ -1,7 +1,11 @@
 :global(html),
 :global(body),
 .app {
-    width: 100%;
+    /* probably unecessary, transitional until layout is refactored */
+    width: 100%; 
     height: 100%;
     margin: 0;
 }
+
+/* TODO: move globally? Safe / side FX, for blocks particularly? */
+* { box-sizing: border-box; }
```

Comment:
We need to figure out the best way to be globals, since this file only is included in the "demo" that you see when you run `npm start`. If we were to include the GUI component in another package, this style wouldn't necessarily be applied.  There is probably something css-modules provides for this kind of thing, we should look into it.